### PR TITLE
Updating to CirrusMD 2.6.0

### PR DESCRIPTION
**Pull request by corclark**
## Description
Updating to CirrusMD SDK 2.6.0 and updating documentation

## Motivation and Context
SDK Updated

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have added tests to cover my changes.
- [x] No tests are required for this change.
- [x] My change requires a documentation change.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 2.6.0 / 2019-3-25
+* **NOTE: This is the last version of the SDK to use the legacy Support Library. 2.7.0 and above will use AndroidX.**
+* Moved to Android 28 and Support Library 28.0.0
+* Now using R8 instead or ProGuard
```
Comment (dlinch) [inline CHANGELOG.md]: typo: `or` => `of`

**Review by ckornher — APPROVED** [review]
Seems like a nice API

File: CHANGELOG.md (hunk 1 above)
Comment (corclark) [reply CHANGELOG.md]: 🌮

File: README.md [hunk 2]
```diff
@@ -121,19 +124,32 @@ The error screen can be shown for several reasons, such as providing an expired
...
+The host application has the ability to fetch and navigate to the list of channels available to the currently authenticated user.
 
-### Support Library Versions
+Calling `CirrusMD.channels()` will return a `List<Pair<String, String>>` which contains the channel information for the current user. These can be used to display the channels with a custom UI anywhere in the host application.
+`Pair.first` represents the channel ID. 
+`Pair.second` represents the channel name.
 
-The current version of the CirrusMD SDK is build with version `27.1.1` of the Android Support Library. 
-If your app requires a different version than what is included, you will need to exclude that package and then import the required packages using your version. However, we do not test against all versions and this could cause problems.
+When a user selects a channel `CirrusMD.navigateToChannel(id: String)` can be called to navigate to the selected channel. The argument is `Pair.first` from the selected item in the list of channels from `CirrusMD.channels()`. Once this is called, the SDK will navigate to the selected channel or will default to the selected when it is displayed.
```
Comment (dlinch) [inline README.md]: should probably say `the selected stream` instead of `the selected`.

**Review by dlinch — APPROVED** [review]
Most of it is gibberish to us non-mobile plebians, but nothing jumped out to me as being particularly confusing besides the two comments.

**Review by rynbyjn — APPROVED** [review]
Words 🔡

**Pull request merged**